### PR TITLE
Translate events

### PR DESCRIPTION
**Pull request by neighborhood999**

### CODE REVIEW

**Comment by jigsawye** [comment]
@neighborhood999 please resolve the conflicts, thanks!

**Comment by neighborhood999** [comment]
@jigsawye ok, plz review and check thank you！

File: events.md [hunk 1]
```diff
@@ -71,12 +71,12 @@ An event class is simply a data container which holds the information related to
         }
     }
 
-As you can see, this event class contains no special logic. It is simply a container for the `Podcast` object that was purchased. The `SerializesModels` trait used by the event will gracefully serialize any Eloquent models if the event object is serialized using PHP's `serialize` function.
+正如你所見的，這個事件類別沒有包含特別的邏輯。它只是一個 `Podcast` 購買物件容器。如果事件物件進行序列化是使用 PHP 的 `serialized` 函式，事件所使用的 `SerializesModels` trait 將會優雅的序列化任何的 Eloquent 模型。
```
Comment (jigsawye) [inline events.md]: It is simply a container for the `Podcast` object that was purchased.

File: events.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-- [Broadcasting Events](#broadcasting-events)
-    - [Configuration](#broadcast-configuration)
-    - [Marking Events For Broadcast](#marking-events-for-broadcast)
-    - [Broadcast Data](#broadcast-data)
-    - [Consuming Event Broadcasts](#consuming-event-broadcasts)
-- [Event Subscribers](#event-subscribers)
+# 事件
+
+- [簡介](#introduction)
+- [註冊事件／監聽者](#registering-events-and-listeners)
```
Comment (jigsawye) [inline events.md]: 註冊事件或監聽者

File: events.md [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
 <a name="introduction"></a>
-## Introduction
+## 簡介
 
-Laravel's events provides a simple observer implementation, allowing you to subscribe and listen for events in your application. Event classes are typically stored in the `app/Events` directory, while their listeners are stored in `app/Listeners`.
+Laravel 事件提供了一個簡單的監聽者實作，允許你在應用程式可以訂閱和監聽事件。事件類別通常被儲存在 `app/Events` 目錄下，而它們的監聽者儲存在 `app/Listeners` 目錄下。
 
 <a name="registering-events-and-listeners"></a>
-## Registering Events / Listeners
+## 註冊事件／監聽者
```
Comment (jigsawye) [inline events.md]: 同上

**Comment by jigsawye** [comment]
@neighborhood999 留 comment 有點囉唆，你可以把我加進你 repository 的 collaborator 嗎？我直接 commit 到你的 repository 上

**Comment by neighborhood999** [comment]
@jigsawye 加入了！再麻煩你了。

**Pull request merged**